editor/src/components/canvas/canvas-strategies/strategies/draw-to-insert-metastrategy.tsx

```diff
@@ -348,6 +349,7 @@ export function drawToInsertStrategyFactory(
 
               return strategyApplicationResult(
                 [insertionCommand.command, reparentCommand, ...optionalWrappingCommand],
+                'rerender-all-elements',
```

Comment:
possibly one to fix here too

Reply:
Added comment about follow up in 38ec355d5.